### ISSUE TITLE
(URGENT) Crashing on Coradoc usage (was reverse_adoc)

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/bipm-data-importer#45]
Job:
* https://github.com/metanorma/bipm-data-outcomes/actions/runs/12193201155/job/34015061866#step:5:318

This points to the need to update this gem to use the new Coradoc API.

**Comment by hmdne** [comment]
It does, but this is not the core issue. The core issue is data update which necessitates code upgrade beyond just updating the gem.

**Comment by ronaldtse** [comment]
Thanks for checking, so we do need to update the gem in a major way. This would be an opportunity to apply Lutaml-model to the outcomes YAML.

**Comment by hmdne** [comment]
I wouldn't say in a major. This is a scraping tool dealing with human submitted data and trying to reach conclusions based on a lot of heuristics, so each time upstream changes we may need to verify and update the heuristics.

From my initial investigation, this tool used to treat CGPM data differently, because they have formatted it differently. That's no longer the case, so we just treat CGPM like every other body.

I have found an issue in Coradoc which I fixed in https://github.com/metanorma/coradoc/pull/142

There are other issues, like the tool fetching French versions for English language.
